code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -762,23 +763,70 @@
 	name = "ratvarian winter coat"
 	icon_state = "coatratvar"
 	item_state = "coatratvar"
+	armor = list("melee" = 30, "bullet" = 20, "laser" = 0,"energy" = 10, "bomb" = 30, "bio" = 10, "rad" = 10, "fire" = 30, "acid" = 30)
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman, /obj/item/toy, /obj/item/storage/fancy/cigarettes, /obj/item/lighter, /obj/item/clockwork/replica_fabricator, /obj/item/clockwork/integration_cog, /obj/item/clockwork/slab, /obj/item/clockwork/weapon/ratvarian_spear)
 	hoodtype = /obj/item/clothing/head/hooded/winterhood/ratvar
+	var/real = TRUE
 
 /obj/item/clothing/head/hooded/winterhood/ratvar
 	icon_state = "winterhood_ratvar"
 	light_range = 3
 	light_power = 1
 	light_color = "#B18B25" //clockwork slab background top color
 
+/obj/item/clothing/suit/hooded/wintercoat/ratvar/equipped(mob/living/user,slot)
+	..()
+	if (slot != SLOT_WEAR_SUIT || !real)
+		return
+	if (is_servant_of_ratvar(user))
+		return
+	else
+		user.dropItemToGround(src)
+		to_chat(user,"<span class='large_brass'>\"Amusing that you think you are fit to wear this.\"</span>")
+		to_chat(user,"<span class='userdanger'>Your skin burns where the coat touched your skin!</span>")
+		user.adjustFireLoss(rand(10,16))
+
 /obj/item/clothing/suit/hooded/wintercoat/narsie
 	name = "narsian winter coat"
 	icon_state = "coatnarsie"
 	item_state = "coatnarsie"
+	armor = list("melee" = 30, "bullet" = 20, "laser" = 30,"energy" = 10, "bomb" = 30, "bio" = 10, "rad" = 10, "fire" = 30, "acid" = 30)

```

Comment:
armor = list("melee" = 30, "bullet" = 20, "laser" = 0,"energy" = 10, "bomb" = 30, "bio" = 10, "rad" = 10, "fire" = 30, "acid" = 30)
So its the same as Clock work coats.

Reply:
cwc armor makes you take extra laser damage

Reply:
Nar armor as always been also weaker to laser then bullet/melee

Reply:
funny thing is I took that directly from empowered armour and scaled it down, it's actually the ratvar one which isn't correct.

Reply:
which is now fixed.